test/integration/oauth_ldap_test.go

```diff
@@ -9,6 +9,9 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/openshift/library-go/pkg/config/helpers"
+	v1 "github.com/openshift/origin/pkg/cmd/server/apis/config/v1"
```

Comment:
I'd prefer we don't alias this as `v1` but more concretely as `configv1`, same applies to other places where `v1` was used.